code/_onclick/hud/rendering/plane_masters/plane_master_subtypes.dm

```diff
@@ -360,12 +360,15 @@
 
 /atom/movable/screen/plane_master/emissive/Initialize(mapload, datum/hud/hud_owner, datum/plane_master_group/home, offset)
 	. = ..()
-	/// Okay, so what we're doing here is making all emissives convert to white for actual emissive masking (i.e. adding light so objects glow)
-	add_relay_to(GET_NEW_PLANE(EMISSIVE_RENDER_PLATE, offset), relay_color = list(1,1,1,0, 1,1,1,0, 1,1,1,0, 0,0,0,1, 0,0,0,0))
-	/// But for the bloom plate we convert only the red color into full white, this way we can have emissives in green channel unaffected by bloom
-	/// which allows us to selectively bloom only a part of our emissives
+	// Okay, so what we're doing here is making all emissives convert to white for actual emissive masking (i.e. adding light so objects glow)
+	add_relay_to(GET_NEW_PLANE(EMISSIVE_RENDER_PLATE, offset), relay_color = list(1,1,1,0, 1,1,1,0, 0,0,0,0, 0,0,0,1, 0,0,0,0))
+	// But for the bloom plate we convert only the red color into full white, this way we can have emissives in green channel unaffected by bloom
+	// which allows us to selectively bloom only a part of our emissives
 	add_relay_to(GET_NEW_PLANE(EMISSIVE_BLOOM_PLATE, offset), relay_color = list(255,255,255,0, 0,0,0,0, 0,0,0,0, 0,0,0,1, 0,0,0,0))
 	add_relay_to(GET_NEW_PLANE(EMISSIVE_BLOOM_MASK_PLATE, offset), relay_color = list(1,1,1,1, 0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0))
+	// Blue channel is dedicated to specular, i.e. our crappy implementation of shiny objects
+	// Alpha must be always maximal as its used to multiply all lighting to add fake shine by increasing existing light
```

Comment:
can you elaborate on the alpha bit here?

Reply:
Because we're mulitiplying, if we don't max out alpha then pixels without emissives or emissive blockers on them will have 0 alpha and thus will get full specular effect as they won't be masked

Reply:
why not use an alpha mask filter? it'd give us more control over how strong speculars are (the little demon in my head also thinks we should increase the specular plane's values by like 1.2 or something to increase the ceiling in dark situations but that'd fuck up the top end so perhaps not)